pkg/apis/eventing/v1alpha1/trigger_types.go

```diff
@@ -74,17 +74,38 @@ type TriggerSpec struct {
 }
 
 type TriggerFilter struct {
-	SourceAndType *TriggerFilterSourceAndType `json:"sourceAndType,omitempty"`
+	// DeprecatedSourceAndType filters events based on exact matches on the
+	// CloudEvents type and source attributes. This field has been replaced by the
+	// Attributes field.
+	//
+	// +optional
+	DeprecatedSourceAndType *TriggerFilterSourceAndType `json:"sourceAndType,omitempty"`
+
+	// Attributes filters events by exact match on event context attributes.
+	// Each key in the map is compared with the equivalent key in the event
+	// context. An event passes the filter if all values are equal to the
+	// specified values.
+	//
+	// Nested context attributes are not supported as keys. Numeric values are
```

Comment:
```suggestion
	// Nested context attributes are not supported as keys. Only string values are supported.
```

(on this line into the next)

Reply:
Done, thanks!